modules/core/src/main/java/org/apache/ignite/internal/processors/affinity/GridAffinityProcessor.java

```diff
@@ -1106,7 +1125,8 @@ public CacheAffinityProxy(String cacheName) {
             }
         }
 
-        /** {@inheritDoc} */
+        /** @deprecated use mapPartitionToPrimaryAndBackupsList instead */
```

Comment:
Please, use `/** {@inheritDoc} */` for overriding method.
The only reason to write a new JavaDoc comment if the implementation has another method contract compared to the parent one.